zipline/src/jniTest/kotlin/app/cash/zipline/ZiplineTest.kt

```diff
@@ -78,16 +77,6 @@ class ZiplineTest {
     assertThat(e.message).isEqualTo("Zipline closed")
   }
 
-  @Test fun inboundServicesReleasedOnClose() = runTest(dispatcher) {
-    zipline.quickJs.evaluate("testing.app.cash.zipline.testing.prepareJsBridges()")
-    val service = JvmEchoService("echo, echo, echo")
-    zipline.bind<EchoService>("helloService", service)
-
-    zipline.close()
-
-    assertThat(zipline.serviceNames).doesNotContain("helloService")
```

Comment:
This test now fails! We actually free the whole Zipline though so we get what we want.